src/ci/github-actions/jobs.yml

```diff
@@ -86,6 +86,8 @@ pr:
     <<: *job-linux-16c
   - image: x86_64-gnu-tools
     <<: *job-linux-16c
+  - image: rfl
+    <<: *job-linux-8c
```

Comment:
The test will eventually be only in the `auto` section, I only kept it in the `pr` section for quick experiments on this PR. Otherwise yes, to comment it out, it is enough to just comment these two lines in the `auto` section.

Reply:
I see. Sounds good, thanks!